src/map.rs

```diff
@@ -195,8 +195,8 @@ pub struct TryInsertError<'a, V> {
 }
 
 impl<'a, V> Display for TryInsertError<'a, V>
-where
-    V: Debug,
+    where
```

Comment:
These changes don't seem right — have you run your code through `rustfmt`? Is your version perhaps old somehow? I get no differences if I run `cargo fmt` locally.